torch/csrc/distributed/rpc/torchscript_functions.cpp

```diff
@@ -17,9 +17,9 @@ c10::intrusive_ptr<c10::ivalue::Future> rpcTorchscript(
     const c10::FunctionSchema& functionSchema,
     std::vector<c10::IValue>& stack,
     const float rpcTimeoutSeconds,
-    const bool asyncFunction) {
+    const bool isAsyncFunction) {
   auto scriptCall = std::make_unique<ScriptCall>(
-      qualifiedName, std::move(stack), asyncFunction);
+      qualifiedName, std::move(stack), isAsyncFunction);
```

Comment:
`isAsyncFunction` should be named to `isAsyncExecution`, will update in the next version.